ingredient/ingredient-event-hub-namespace/src/index.ts

```diff
@@ -8,3 +8,5 @@ exports.pluginNS = "@azbake/ingredient-event-hub-namespace" //name of the ingred
 /* comment out these entries if you are not including a set of expression functions*/
 exports.functions = EventHubNamespaceUtils
 exports.functionsNS = "eventhubnamespace" //bake.yaml expressions can access your functions via "myutils.my_function()"
+
+export {EventHubNamespaceUtils}
```

Comment:
don't do this. Will give odd results with how bake uses ingredients

Reply:
See other comment